packages/icons/icons.json

```
@@ -4625,5 +4625,12 @@
         "tags": "blockage, sign, road, stop",
         "group": "miscellaneous",
         "codepoint": 62357
+    },
+    {
+        "displayName": "Annoucement",
+        "iconName": "Annoucement",
```

"Annoucement" should be lowercased here to match the casing of SVG filenames:
```suggestion
        "iconName": "annoucement",
```